bundles/org.openhab.binding.ftpupload/src/main/java/org/openhab/binding/ftpupload/internal/handler/FtpUploadHandler.java

```diff
@@ -67,6 +67,11 @@ public void initialize() {
         configuration = getConfigAs(FtpUploadConfig.class);
         logger.debug("Using configuration: {}", configuration.toString());
 
+        if (configuration.userName.isBlank() || configuration.password.isBlank()) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "Username and password must be configured");
```

Comment:
It would be nice to add this string as I18N, so it can be localized.

Reply:
Done